asyncnsq/tcp/reader.py

```diff
@@ -68,6 +68,7 @@ def __init__(self, nsqd_tcp_addresses=None, lookupd_http_addresses=None,
 
         self._rdy_control = None
         self._max_in_flight = max_in_flight
+        self._num_readers = 0
```

Comment:
will explain a little for self._num_readers usage?

Reply:
this is a reference counter to make sure the 'unsubscribe' functions could close safely.